rest-api/data_gen/fake_participant_generator.py

```diff
@@ -138,6 +127,12 @@ def __init__(self, client, use_local_files=None):
     self._setup_questionnaires()
     self._min_birth_date = self._now - datetime.timedelta(days=_MAX_PARTICIPANT_AGE * 365)
     self._max_days_for_birth_date = 365 * (_MAX_PARTICIPANT_AGE - _MIN_PARTICIPANT_AGE)
+    self._force_measurement = False
+
+    # 5% of participants withdraw from the study
```

Comment:
nit: this probably shouldn't say 5% anymore, since that's just the default value now